src/fidesops/service/connectors/query_config.py

```diff
@@ -396,6 +397,72 @@ def dry_run_query(self) -> Optional[str]:
         return None
 
 
+class MicrosoftSQLServerQueryConfig(SQLQueryConfig):
+    """
+    Generates SQL valid for SQLServer. This way of building queries should also work for every other connector,
+    but SQLServer is separated due to increased code complexity for building queries
+    """
+
+    def format_clause_for_query(
+            self,
+            field_name: str,
+            operator: str,
+            operand: str
+    ) -> str:
+        """Returns clauses in a format they can be added into SQL queries."""
+        if operator == "IN":
+            return f"{field_name} IN ({operand})"
+        return super().format_clause_for_query(field_name, operator, operand)
+
+    def generate_query(  # pylint: disable=R0914
+        self,
+        input_data: Dict[str, List[Any]],
+        policy: Optional[Policy] = None,
+    ) -> Optional[TextClause]:
+        """Generate a retrieval query"""
+
+        filtered_data = self.typed_filtered_values(input_data)
+
+        if filtered_data:
+            clauses = []
+            query_data: Dict[str, Tuple[Any, ...]] = {}
+            formatted_fields = self.format_fields_for_query(self.fields)
+            field_list = ",".join(formatted_fields)
+
+            for field_name, data in filtered_data.items():
```

Comment:
Can you update the `field_name` variable to be `string_path`? It's a new change to the other `generate_query` methods to reflect that this can potentially be a dot separated name, representing a nested field.

Reply:
ah yes!

Reply:
⭐

Reply:
I've renamed other occurrences I saw in `query_config` as well

Reply:
thank you! still lots to do here around nested objects - these queries aren't actually supported yet, so a lot of these places still subject to change over the next few weeks!